.github/workflows/push.yml

```
@@ -0,0 +1,53 @@
+          clean: false
+      - name: Deploy the global.json file
+        run: cp ${{ github.workspace }}/global.json ${{ env.pathToCi }}/
+      - name: Load cached data from previous runs
+        uses: actions/cache@v4
+        with:
+          path: |
+            ${{ env.pathToCi }}/.nuke/temp
+            ~/.nuget/packages
+          key: ${{ runner.os }}-${{ hashFiles('**/global.json', '**/*.csproj') }}
+      - name: Build Package Version Listing with Nuke
+        run: ${{ env.pathToCi }}/build.cmd BuildRepoListing --root ${{ env.pathToCi }} --list-publish-directory $GITHUB_WORKSPACE/${{ env.listPublishDirectory }} --current-package-name ${{ vars.PACKAGE_NAME }}
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:100: Double quote to prevent globbing and word splitting

(shellcheck)